### DISCUSSION TITLE
Change the item icon on card layout

### DISCUSSION THREAD

**Discussion by Guiqft** [directus/directus#3660] [Feature Requests]
I've a custom display for show me a image (the column value is the image url). 
When I change the layout view for "Card", I wanna be able to select my custom display to be the item preview icon. 
How can I do that?

**Comment by benhaynes** [comment]
That isn't possible as of now. The Displays are shown _inline_, so they can be in the title/subtitle of the cards. The card image themselves can only pull from an image field. It would be a good enhancement to allow pointing the card image to a URL string though! Is that what you were asking for?

**Reply by Guiqft** [reply]
Yeah, if this was possible would help me in my case, since the column value it's a image URL string.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
